lsm-perf.py

```diff
@@ -164,7 +210,18 @@ def __construct_qemu_args(kernel_path, filesystem_img_path):
             '-serial', 'mon:stdio',
             '-kernel', '%s' % kernel_path,
             '-name', 'lsm_perf_vm,debug-threads=on'
-        ]
+        ] + qemu_opt
+
+    @staticmethod
+    def __qemu_affinity_setup(qemu_pid, cpu_alloc):
+        """Run qemu_affinity.py to allocate CPUs based on the CPU_Allocation"""
+        system_affinities = ('-p %(sys)d -i *:%(sys)d -q %(sys)d -w *:%(sys)d'
+                             % {'sys': cpu_alloc.qemu_sys}).split(' ')
+        kvm_affinities = ['-k', str(cpu_alloc.host_kvm0),
+                                str(cpu_alloc.host_kvm1)]
+        args = system_affinities + kvm_affinities + ['--', str(qemu_pid)]
+        cmd = plumbum.cmd.sudo['python3'][QEMU_AFFINITY_PATH][args]
```

Comment:
Is this a step you only have to do once? If so, given that it requires sudo it might just be better as a step that the user has to do separately, rather than having the main script prompt for a password so it needs to be babysat

Reply:
It is done each time we start a VM, so `#rounds * #kernel`. In practice (except if you changed the default config of your shell), it will ask your password only the first time and remember it for the other times. Just like `sudo echo 1; sudo echo 1` asks your password only once. 
We might want to get ride of using `sudo` all together, this will requires some modification to qemu-affinity.